arib/app/auth/auth-http.service.ts

```diff
@@ -23,16 +24,14 @@ export class AuthHttp {
     get(url: string) {
         let headers = new Headers();
         this.createAuthorizationHeader(headers);
-        return this.http.get(this.config.apiURL + "/api/" + url, {
-            headers: headers
-        });
+        return this.http.get(this.config.apiURL + "/api/" + url, { headers })
+            .map((res: Response) => res.json());
```

Comment:
The errors are unchanged, so that works like before. Only the succesful responses are now automatically parsed as JSON, which I think is what every consumer of this method wants.